build/packages/rpm/server/SPECS/victoriametrics.spec

```diff
@@ -1,3 +1,5 @@
+%undefine _missing_build_ids_terminate_build
```

Comment:
Is this change related to the PR?

Reply:
Yes and no. If you look at the logs, the pipeline was failing without this. It's the last component that does not have this statement, all others do already.